.github/workflows/build.yaml

```
@@ -0,0 +1,25 @@
+name: build
```

Or maybe "Test" as we are not actually building anything here? 😃 
```suggestion
name: Test
```